packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -80,6 +82,9 @@ export const BackdropView = styled.div`
 	transition: transform ${ CONFIG.transitionDurationFast } ease;
 	${ reduceMotion( 'transition' ) }
 	z-index: 1;
+	// Windows High Contrast mode will show this outline, but not the box-shadow.
+	outline: 2px solid transparent;
+	outline-offset: -3px;
```

Comment:
At `-2px`, the space between outlines seemed too narrow:

![offset-2-component](https://github.com/WordPress/gutenberg/assets/54422211/d782da6d-927a-491e-a880-a13ed44d0d47)

![offset-2-block-editor](https://github.com/WordPress/gutenberg/assets/54422211/724745c8-0be3-4835-9d79-975b6f994a63)